cgo_libc_static/README.md

```diff
@@ -152,21 +152,37 @@ func (cn *conn) ssl(o values) {
 }
 ```
 
-It's relatively easy to guess this in this heavily truncated version, but there's actually a successful call to `user.Current()` from `userCurrent()` (marked with `!!!`). We only saw this after adding an `fmt.Println()` in `user.Current()` and wondered why that printed more than we expected. So, that's weird - the crash is either random<sup>2</sup> or it depends on something else happening before it.
+It's relatively easy to guess this in this heavily truncated version, but
+there's actually a successful call to `user.Current()` from `userCurrent()`
+(marked with `!!!`). We only saw this after adding an `fmt.Println()` in
```

Comment:
I'm "eff-em-tee", but apparently the majority is "fumt", so changed to `a`.

On Tue, Dec 15, 2015 at 11:16 AM Peter Mattis <notifications@github.com>
wrote:

> In cgo_libc_static/README.md
> <https://github.com/tschottdorf/goplay/pull/2#discussion_r47658348>:
>
> > @@ -152,21 +152,37 @@ func (cn *conn) ssl(o values) {
> >  }
> >  ```
> >
> > -It's relatively easy to guess this in this heavily truncated version, but there's actually a successful call to `user.Current()` from `userCurrent()` (marked with `!!!`). We only saw this after adding an `fmt.Println()` in `user.Current()` and wondered why that printed more than we expected. So, that's weird - the crash is either random<sup>2</sup> or it depends on something else happening before it.
> > +It's relatively easy to guess this in this heavily truncated version, but
> > +there's actually a successful call to `user.Current()` from `userCurrent()`
> > +(marked with `!!!`). We only saw this after adding an `fmt.Println()` in
>
> Should that be a fmt, or is an fmt correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tschottdorf/goplay/pull/2/files#r47658348>.
>